pkg/controller/bastion/actuator_reconcile.go

```diff
@@ -50,7 +50,7 @@ func (be *bastionEndpoints) Ready() bool {
 	return be != nil && IngressReady(be.private) && IngressReady(be.public)
 }
 
-func (a *actuator) Reconcile(ctx context.Context, bastion *extensionsv1alpha1.Bastion, cluster *controller.Cluster) error {
+func (a *actuator) Reconcile(ctx context.Context, _ logr.Logger, bastion *extensionsv1alpha1.Bastion, cluster *controller.Cluster) error {
 	logger := a.logger.WithValues("bastion", client.ObjectKeyFromObject(bastion), "operation", "reconcile")
```

Comment:
unlike `calico` extension, here logger is used in some functions, please adapt accordingly.
Ref: https://github.com/gardener/gardener-extension-provider-aws/pull/585/commits/78f27287765291e1b6c3481774c5e5e7040f5d8f

Reply:
PR feedback d5aa79f6